src/pages/singer/Docs.vue

```diff
@@ -1,358 +1,371 @@
-      database’s binary log files to detect records that have been inserted, updated, or deleted.
-      Supporting this capability, where applicable, means that targets have the information it needs
-      to delete records in the destination.
-
-      <p class="text-lg">Test</p>
-      This capability is still in development, but the goal is to make it easy for connector
-      developers to test locally without having to make a connection to the source systems. This
-      might be accomplished by implementing utilities that can generate and save test data or make
-      saving and moving real HTTP requests for tests easier. Join the issue conversation here!
-
-      <h3>Target Specific</h3>
-      <p class="text-lg">Soft Delete</p>
-      <p>
-        Targets that support this capability implement logic that soft deletes records in the
-        destination, usually by populating a deleted_at timestamp field. The two common techniques
-        for handling deletes in Singer are by using the ACTIVATE_VERSION message or using log-based
-        replication on a database that supports it. Targets commonly implement both soft and hard
-        deletes, in which case the behavior is configurable in the target’s config settings.
-      </p>
-
-      <p class="text-lg">Hard Delete</p>
-      <p>
-        Similar to the Soft Deletes capability, targets that support the hard delete capability
-        implement logic that removes records from the destination when they receive a notification
-        (via ACTIVATE_VERSION or log-based replication) that a record was deleted from the source
-        system. Targets commonly implement both soft and hard deletes, in which case the behavior is
-        configurable in the target’s config settings.
-      </p>
-
-      <p class="text-lg">Datatype Failsafe</p>
-      <p>
-        This capability means that the target has a failsafe data type (e.g. string) to ensure safe
-        handling of unparsable or unforseen types. It’s common to find targets in the Singer
-        ecosystem that have a set list of supported types and will throw an exception if they
-        receive anything unexpected. With this capability supported you can feel comfortable that
-        exceptions will not be raised for unsupported types.
-      </p>
-
-      <p class="text-lg">Record Flattening</p>
-      <p>
-        It’s common for taps to output nested json data which needs to be flattened by the target in
-        order to be loaded into a columnar format destinations. For example, the following is a
-        nested json record:
-      </p>
-      <pre><code>
-{
-  "col_a": {
-    "subcol_1": 1,
-    "subcol_2": 2
+    <div class="grid w-full">
+      <div class="h-7 max-h-7"></div>
+      <div class=" bg-white border text-black w-full prose lg:prose-xl">
+        ## Test test
+
+        This is markdown, `kinda`.
+
+        TODO: See if we can render from raw markdown here, or from a file. (Doesn't render as of now.)
```

Comment:
- [ ] TODO: @aaronsteers

Reply:
Resolved in 3012d832dbbce3d33564479baf9b0e0909fdc3c8 and aee1f4cead0db1de8d7d8d26d01337642889592f.